infoschema/cache.go

```diff
@@ -40,9 +50,9 @@ func NewCache(capcity int) *InfoCache {
 func (h *InfoCache) GetLatest() InfoSchema {
 	h.mu.RLock()
 	defer h.mu.RUnlock()
-	metrics.InfoCacheCounters.WithLabelValues("get", "latest").Inc()
+	getLatestCounter.Inc()
```

Comment:
How much performance degradation will WithLabelValues bring?

Reply:
Several seconds, though it is not a big deal in the total performance degradation time. The degradation disappears after removing `metrics`.